capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @decoration.correct_name.upcase
```

Comment:
Kindly use the `capitalize` method to capitalize only the first letter of the name. `upcase` returns a copy of the string with all the lowercase letters replaced with their uppercase counterparts.